modules/core/package.json

```diff
@@ -39,5 +39,8 @@
     "math.gl": "^3.4.2",
     "mjolnir.js": "^2.5.0",
     "probe.gl": "^3.3.1"
+  },
+  "devDependencies": {
```

Comment:
Longer term, I wonder if this coild be imported and re-exported from the mjolnir.js module instead, as that is where the hammerjs dependency is.

Reply:
seems like ok. but mjolnir.js do not have type definition file ....
import form hammerjs , has types file ✓
import form mjolnir.js, no types file ×
or just add a standalone d.ts (not recommend. if code changed, need to change it every time)